README.md

```diff
@@ -1053,7 +1053,7 @@ Table of Contents
   * [freenom](https://freenom.com) — Free .tk,.ml,.ga,.cf and .gq domain.
   * [eu.org](https://nic.eu.org) — Free eu.org domain. Request is usually approved in 14 days.
   * [js.org](https://js.org) - Free js.org subdomains for GitHub Pages for the JavaScript community.
-  * [js.cool](https://github.com/willin/js-cool) - Free js.cool subdomains for GitHub Pages and Vercel.
+  * [js.cool](https://github.com/willin/js.cool) - Free js.cool subdomains for GitHub Pages and Vercel.
```

Comment:
It’s nice and all but I can’t really add it as this is just too restrictive. 


“ Your site must have a Chinese translation edition”

Reply:
Are these actually CNAMEs as stated? It doesn’t seem that way to me. It seems like a system of redirects rather than CNAME?

Reply:
- https://retrounhash.js.cool/
- https://leader.js.cool/
- https://xrkffgg.js.cool/
- and so on

Reply:
Yes I saw and none of these are CNAMEs they resolve to the same IP.

Reply:
Compare with for example js.org - that one makes actual CNAME dns entries. This one does not do  that.

Reply:
To be clear: I am not listing a redirect based service.

Reply:
<img width="873" alt="image" src="https://user-images.githubusercontent.com/1890238/164391921-037b974c-86f9-40be-a6d3-3fc288123b58.png">


https://github.com/willin/js.cool/blob/master/main/active_cname.js

Reply:
perhaps because proxied?

Reply:
Yes.

Reply:
Others like js.org are not proxies

Reply:
same with js.org, it depends on users' willing: https://github.com/js-org/js.org/blob/master/cnames_active.js#L17-L22

Reply:
Suggest you show this in similar comments in readme or here and I would need an example that works

Reply:
and it's almost the same implementation.